src/modules/Sticky/Sticky.js

```diff
@@ -199,13 +200,17 @@ export default class Sticky extends Component {
   }
 
   computeStyle() {
-    const { bottom, sticky, top } = this.state
-
+    const { bottom, bound, sticky, top } = this.state
     if (!sticky) return {}
+    if (bound) {
+      return {
+        bottom: 0,
+        width: this.triggerRect.width,
+      }
+    }
```

Comment:
This condition is strange, I think that we should just have `bottom: null` from the component's state there.